examples_tests/13.BRDF_Explorer/BRDFExplorerApp.cpp

```diff
@@ -340,54 +340,69 @@ float applyChainRule(in vec2 h_gradient, in vec2 dUVd_)
 }
 
 // Calculate the surface normal using the uv-space gradient (dhdu, dhdv)
-vec3 calculateSurfaceNormal(in vec3 position, in vec2 uv, in vec3 normal, in vec2 h_gradient)
+vec3 calculateSurfaceNormal(in vec3 normal, in vec2 h_gradient, in vec3 dpdx, in vec3 dpdy, in vec4 dUVd_)
```

Comment:
why did you feel the need to pack the texture coord derivatives in a vec4 ?

Reply:
less parameters

Reply:
its confusing, also it won't help anything.